sdk/eventhub/Azure.Messaging.EventHubs/tests/Compatibility/TrackOneExceptionExtensionsTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Messaging.EventHubs.Compatibility;
+using Azure.Messaging.EventHubs.Errors;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="TrackOneExceptionExtensions" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    [Parallelizable(ParallelScope.Children)]
+    public class TrackOneExceptionExtensionsTests
+    {
+        /// <summary>
+        ///   The set of test cases for the well-known types derrived from the <see cref="EventHubsException" />
```

Comment:
nit: derived?

There are a few other occurrences in the same file.

Reply:
Thank you... did a global replace.